charts/images.yaml

```diff
@@ -70,8 +70,8 @@ images:
   tag: v0.17.0
 - name: grafana
   sourceRepository: github.com/grafana/grafana
-  repository: grafana/grafana
-  tag: "5.4.2"
+  repository: eu.gcr.io/gardener-project/registry.hub.docker.com/grafana/grafana
```

Comment:
As discussed: We probably want to keep using the "normal" grafana image?!

Reply:
yep done.